rotkehlchen/chain/ethereum/modules/balancer/utils.py

```diff
@@ -243,6 +246,7 @@ def deserialize_pool_share(
         protocol='balancer',
         decimals=18,  # All BPT tokens have 18 decimals
         underlying_tokens=pool_tokens,
+        seen=TokenSeenAt(description='Balancer pools query'),
```

Comment:
For consistency with the other descriptions should it be

```suggestion
        seen=TokenSeenAt(description='Querying balancer pools'),
```

?

Reply:
Will see in a subsequent PR if it makes sense!